.changeset/chatty-onions-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-application': patch
+'@finos/legend-manual-tests': patch
+'@finos/legend-studio': patch
+---
```

Comment:
What I meant is you need 2 changesets, 1 with no messages, 1 with a message and the package you actually put the fix on, i.e. `@finos/legend-studio`